Added a link to live viewing from monitor event viewer

IgorA100 opened the pull request:
1. When viewing events, there is no way to go to a live view of the monitor in real time in one click.
2. But, there are two links to go to the monitor settings.

It was decided to replace one of the links with a transition to live viewing.

![P1040680](https://github.com/ZoneMinder/zoneminder/assets/5006170/aa3881e2-9804-4bce-8893-defa0b1099ea)

connortechnology commented:
But you removed the link to go to monitor settings.  How about instead of this, we use icons and have on for settings and one for live view?

IgorA100 commented:
I replaced ONE link out of two!
The second one remains. Using it you can easily get into the monitor settings.
![12r](https://github.com/ZoneMinder/zoneminder/assets/5006170/e00134dc-9352-42f9-b09a-970ee329ffba)

I thought about adding a button, but I didn’t like this idea because... it will be a redundant element.
There is already a clickable place. Why have two links to the same action next to each other? And I decided to replace one of the links. This way, unnecessary elements are not added and the functionality is expanded.

connortechnology commented:
Ok, but it's not very good UI.  I actually think the two lines should be merged.  It isn't intuitive that clicking on one goes one place and the other a different place, which is why I think we should use icons.

IgorA100 commented:
Ok, since you think it would be better to add another button and remove one duplicate line, I can do that.
I suggest making a button for live viewing like this: https://fontawesomeicons.com/television
or this one: https://fontawesomeicons.com/video-camera
Or offer your own option.
And in what place is it best to add it?

connortechnology commented:
That icon looks fine.  We include both material icons and font awesome... it would be good at some point to standardize on one of them and remove the other.  At this time I don't know which is better, although I tend to gravitate towards material icons, but then I often don't find what I need.  I would just insert the buttons on the line with the Monitor Id/Name.

IgorA100 commented:
I changed the design.
I moved name & Id of the monitor to another place, similar to live viewing. I think it will be better this way.
I would add another button to go to view all events. Currently there is only a button at the top to view all frames for the event.
![21](https://github.com/ZoneMinder/zoneminder/assets/5006170/e3b72a10-af5e-4cc6-9a43-23748f3566f1)
Yes, I saw that different sets of icons are used.
I agree that ideally you should use one set. But I can’t say anything concrete yet.
I'll see if it's possible to give up one of the sets.

IgorA100 commented:
For example like this:
![1112](https://github.com/ZoneMinder/zoneminder/assets/5006170/7dc41234-7a5d-4351-9bc2-d7e929745005)

IgorA100 commented:
@connortechnology 
If you like the picture, I'll make changes to the code.

connortechnology commented:
It looks great, but I want you to keep the Id and Name before the buttons.

IgorA100 commented:
> It looks great, but I want you to keep the Id and Name before the buttons.

No problem.
![121](https://github.com/ZoneMinder/zoneminder/assets/5006170/e4136948-1515-42f2-8bda-ebe9d2c162a4)

But it won’t be pretty, especially if the monitor name is long :(
Why do this? The monitor ID and its name are normally visible under the video.
![210](https://github.com/ZoneMinder/zoneminder/assets/5006170/adcf2063-0bc0-4c9e-a999-9f2fc6e6c1a6)

connortechnology commented:
THere is enough space there for it to fit.  I think we just need a little css magic to make it layout better.   honestly I would remove the monitor info at the bottom.  
My feelings on the matter are simply that I am used to looking there for the name, I want to know which monitor it is.

IgorA100 commented:
> THere is enough space there for it to fit. I think we just need a little css magic to make it layout better. 

css is not a magician :)
Yes, with the current monitor name and high resolution there is no problem displaying everything in one line.
But, moving icons to a new line will happen in the following case:
- long monitor name
- low screen resolution
There is no way to fix this with css.
Or make two lines at once (one with ID and name, the second with icons).

IgorA100 commented:
In the works, a fix will be coming soon.
A lot of work. time is short.

IgorA100 commented:
I finished work
Now like this:
![+1](https://github.com/ZoneMinder/zoneminder/assets/5006170/5779dc4d-ca65-450b-993e-1e0850c82577)
![+2](https://github.com/ZoneMinder/zoneminder/assets/5006170/9f1dcc8a-66e6-4972-a8d5-b4e842a8fb53)
![+3](https://github.com/ZoneMinder/zoneminder/assets/5006170/336cac71-4bce-4edc-866f-08d9a6c2b7eb)

Now everything scales correctly depending on the screen size.

IgorA100 commented:
If you have any comments, I am ready to discuss.
But I think it turned out not bad.

connortechnology commented:
Good work, looks great.  I was wanting this feature just the other day.

IgorA100 commented:
@connortechnology 
Thank you!
If I find time I will make changes (using responsive Bootstrap) for live view. There are some things that are wrong.
